tests/run_tests.sh

```diff
@@ -90,6 +90,8 @@ function setup_environment()
     export ANSIBLE_CONFIG=${BASE_PATH}/ansible
     export ANSIBLE_LIBRARY=${BASE_PATH}/ansible/library/
     export ANSIBLE_CONNECTION_PLUGINS=${BASE_PATH}/ansible/plugins/connection
+
+    rm -fr ${BASE_PATH}/tests/_cache
```

Comment:
I'm thinking about add logs to the decorator `cached` to make test user aware of where the facts come from(cached or function call).